src/static/AppStatic.res

```diff
@@ -0,0 +1,3 @@
+let overallHeader = "Expense Tracker"->React.string
```

Comment:
Don't use `React.string` here. Only keep the static values here. Applicable in all the static files.